src/actions/layout.js

```diff
@@ -10,4 +17,51 @@ const closeSettings = () => dispatch =>
     type: CLOSE_SETTINGS,
   });
 
+export const getSpaceTree = () => async (dispatch, getState) => {
```

Comment:
Maybe a good thing would be to add a simple test with the fake API so that we can ensure that the functions are acting as expected.